packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -401,7 +459,318 @@ test.serial(
+    getVoteId(6),
+    offerIds.vote.incoming,
+  );
+  for (const w of newCommittee.slice(0, 2)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(6), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('no oracle invitation should exist before vote passing');
+  const oracleInvitation =
+    await governanceDriver.ecMembers[0].findOracleInvitation();
+  t.is(oracleInvitation, undefined);
+
+  t.log('Checking params before passing proposal');
+  const reserveParams = getVstorageData(reserveParamsKey);
+  t.is(reserveParams.totalFeeBurned.value, 0n);
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  t.assert(lastOutcome.outcome === 'win');
+
+  const reserveParamsPostUpdate = getVstorageData(reserveParamsKey);
+  t.is(reserveParamsPostUpdate.totalFeeBurned.value, 1000n);
+});
+
+test.serial('EC can govern psm parameter', async t => {
+  const { storage, advanceTimeBy, getVstorageData, governanceDriver } =
+    t.context;
+  const newCommittee = governanceDriver.ecMembers.slice(0, 3);
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { IST } = agoricNamesRemotes.brand;
+
+  const psmInstances = Object.keys(agoricNamesRemotes.instance)
+    .map(instance => {
+      const regex = /^psm-IST-(?<brand>.*)$/;
+      return regex.exec(instance);
+    })
+    .filter(instance => instance !== null);
+
+  await null;
+  for (const instance of psmInstances) {
+    const brand = instance.groups?.brand;
+    const instanceName = instance[0];
+    t.log('Proposing question using new charter invitation for', instanceName);
+    await governanceDriver.proposeParams(
+      agoricNamesRemotes.instance[instanceName],
+      { MintLimit: { brand: IST, value: 100_000_000n } },
+      { paramPath: { key: 'governedParams' } },
+      newCommittee[0],
+      getQuestionId(instanceName),
+      offerIds.propose.incoming,
+    );
+
+    t.like(newCommittee[0].getLatestUpdateRecord(), {
+      status: { id: getQuestionId(instanceName), numWantsSatisfied: 1 },
+    });
+
+    t.log('Voting on question using first 2 wallets');
+    await governanceDriver.enactLatestProposal(
+      newCommittee.slice(0, 2),
+      getVoteId(instanceName),
+      offerIds.vote.incoming,
+    );
+    for (const w of newCommittee.slice(0, 2)) {
+      t.like(w.getLatestUpdateRecord(), {
+        status: { id: getVoteId(instanceName), numWantsSatisfied: 1 },
+      });
+    }
+
+    t.log('Waiting for period to end');
+    await advanceTimeBy(1, 'minutes');
+
+    t.log('Verifying outcome');
+    const lastOutcome = await governanceDriver.getLatestOutcome();
+    const psmParams = getVstorageData(getPsmKey(brand));
+    t.deepEqual(psmParams.current.MintLimit.value.value, 100_000_000n);
+    t.assert(lastOutcome.outcome === 'win');
+  }
+});
+
+test.serial('EC can govern price feed parameter', async t => {
```

Comment:
this case isn't about a parameter.

```suggestion
test.serial('EC can make calls to price feed governed APIs', async t => {
```

_not a critical correctness issue, but misleading_